packages/swc-helpers/src/_object_spread_props.js

```diff
@@ -0,0 +1,30 @@
+function ownKeys(object, enumerableOnly) {
+  var keys = Object.keys(object);
+  if (Object.getOwnPropertySymbols) {
+    var symbols = Object.getOwnPropertySymbols(object);
+    if (enumerableOnly) {
+      symbols = symbols.filter(function (sym) {
+        return Object.getOwnPropertyDescriptor(object, sym).enumerable;
+      });
+    }
+    keys.push.apply(keys, symbols);
+  }
+  return keys;
+}
+
+export default function _objectSpreadProps(target, source) {
+  source = source != null ? source : {}
+  if (Object.getOwnPropertyDescriptors) {
```

Comment:
But well, this is supported by ie 11 so it's fine.

Reply:
https://caniuse.com/?search=getOwnPropertyDescriptor

Reply:
Do we really need this check?

Reply:
It's `getOwnPropertyDescriptors` and IE 11 doesn't support that.

Reply:
Ah you are right, thank you!